components/brave_news/browser/resources/feed/Article.tsx

```diff
@@ -31,16 +31,23 @@ const ArticleImage = styled.img`
 
   border-radius: 6px;
 `
-export const openArticle = (article: FeedItemMetadata) => window.open(article.url.url, '_blank', 'noopener noreferrer')
+export const openArticle = (article: FeedItemMetadata) => {
+  const href = article.url.url
+  validateScheme(href)
+  window.location.href = href
```

Comment:
cc @thypon I think the github-actions bot should flag this too 

```
window.location.href = `javascript:alert('hello')` 
```

will quite happily run JavaScript, if your CSP allows.

Reply:
Then again, maybe it's a losing battle:
535 ways to navigate in JavaScript (and it misses a bunch)
https://www.phpied.com/files/location-location/location-location.html